MonoGame.Framework/Media/Song.NVorbis.cs

```diff
@@ -4,19 +4,29 @@
 
 using System;
 using System.IO;
-using Microsoft.Xna.Framework.Audio;
-using Tao.Sdl;
+using MonoGame.Utilities;
+using OpenTK.Audio;
 
 namespace Microsoft.Xna.Framework.Media
 {
     public sealed partial class Song : IEquatable<Song>, IDisposable
     {
-        private IntPtr _audioData;
-        private int _volume = 128; // in SDL units from 0 to 128
+        private AudioContext acontext;
+        private OggStreamer oggstreamer;
```

Comment:
coding note 1, the above 2 variables are here only for testing. I was gonna move them to OpenTKGamePlatform since only 1 instance of them is needed for playing stuff.